### ISSUE TITLE
[DatePicker]: Calendar doesn't take focus on toggle when placed in <label>

### ISSUE THREAD

**Issue by ggkrustev** [telerik/kendo-angular#1295]
## I'm submitting a...
* Bug report

## Current behavior
The component opens the calendar on toggle click, but doesn't moves the focus to the popup.

## Expected behavior
Open the calendar and move the focus to it.

## Minimal reproduction of the problem with instructions
1. Open https://plnkr.co/edit/PT26urCdNhyMkkbOE7Lb?p=preview in **Firefox**
2. Click the icon to open the Calendar
3. Notice that the input is still focused.

## What is the motivation or use case for changing the behavior?
Ensure a consistent behavior when use both <label> and <label for="" /> controls.

## Environment

Package versions:
```
├── @angular/animations@4.4.6
├── @angular/common@4.4.6
├── @angular/compiler@4.4.6
├── @angular/compiler-cli@4.4.6
├── @angular/core@4.4.6
├── @angular/forms@4.4.6
├── @angular/http@4.4.6
├── @angular/platform-browser@4.4.6
├── @angular/platform-browser-dynamic@4.4.6
├── @angular/platform-server@4.4.6
├── @angular/router@4.4.6
├── @progress/kendo-angular-intl@1.3.0
├── @progress/kendo-angular-l10n@1.0.5
├── @progress/kendo-angular-popup@2.0.0
├── @progress/kendo-theme-default@2.47.4
├── @types/jest@21.1.10
├── @types/zone.js@0.0.27
├── core-js@2.5.3
├── ghooks@1.3.2
├── gulp@3.9.1
├── jest@21.2.1
├── rxjs@5.5.6
├── tslib@1.9.0
├── tslint@5.9.1
├── typescript@2.5.3
└── zone.js@0.8.12
```

Browser:
- Firefox version

System:
- TypeScript version: 2.5.3
- Node version: 8
- Platform:  Mac

Reported in TicketID: 1152130

**Comment by dimitar-pechev** [comment]
Fixed in the latest [dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the `@progress/kendo-angular-dateinputs` package (**4.0.1-dev.201907111211**).

**Comment by dimitar-pechev** [comment]
Shipped with **v4.0.1**.

**Comment by rdraisinghani** [comment]
Is there any workaround for this?

**Comment by dimitar-pechev** [comment]
@rdraisinghani, there's no need for a workaround - the bug has been fixed in **v4.0.1** of the **DateInputs** package. Here's a [StackBlitz example](https://stackblitz.com/edit/angular-yfzm1r-pinckx) with the latest version of the package with the corrected behavior - clicking on the calendar icon now focuses the opened calendar just as expected.

Just update the `@progress/kendo-angular-dateinputs` package to its latest version to pull in the bug fix.
